### PR TITLE
add TokenSale to deploy script

### PR DESCRIPTION
**Pull request by MrToph**
# changelog
- add token sale to deploy & verify scripts

### CODE REVIEW

File: scripts/deploy.ts [hunk 1]
```diff
@@ -77,6 +81,21 @@ task('deploy', 'deploy contracts').setAction(async (taskArgs, hre) => {
   await governor.deployed();
   console.log(`governor address: ${governor.address}`);
 
+  console.log(`deploying tokensale...`);
+  const TokenSaleFactory = (await hre.ethers.getContractFactory('TokenSale')) as TokenSale__factory;
+  tokenSale = await TokenSaleFactory.deploy(
+    token.address,
+    config.TOKEN_SALE_USDC,
```
Comment (HickupHH3) [inline scripts/deploy.ts:88]: tokenIn = USDC
tokenOut = ARENA

File: scripts/verify.ts [hunk 2]
```diff
@@ -32,10 +34,19 @@ task('verifyContracts', 'verify deployed contracts')
       new Date(config.CLAIM_END_DATE).getTime() / 1000,
       config.VEST_DURATION,
     ]);
-
     await verifyContract(hre, revokableTokenLock.address, [token.address, addresses['deployer']]);
     await verifyContract(hre, timelock.address, [config.TIMELOCK_DELAY, [], []]);
     await verifyContract(hre, governor.address, [token.address, timelock.address]);
+    await verifyContract(hre, tokenSale.address, [
+      token.address,
+      config.TOKEN_SALE_USDC,
```
Comment (HickupHH3) [inline scripts/verify.ts:42]: here too

**Review by HickupHH3 — CHANGES_REQUESTED** [review]

File: scripts/deploy.ts (hunk 1 above)
Comment (MrToph) [reply scripts/deploy.ts]: thanks, fixed

File: scripts/verify.ts (hunk 2 above)
Comment (MrToph) [reply scripts/verify.ts]: fixed

**Review by tabshaikh — APPROVED** [review]
LGTM

**Review by HickupHH3 — APPROVED** [review]
LGTM.

**Pull request merged**